src/gui/attributetable/qgsorganizetablecolumnsdialog.cpp

```diff
@@ -142,3 +144,25 @@ void QgsOrganizeTableColumnsDialog::hideAll()
     mFieldsList->item( i )->setCheckState( Qt::Unchecked );
   }
 }
+
+void QgsOrganizeTableColumnsDialog::toggleSelection()
+{
+  for ( int i = 0; i < mFieldsList->count() ; i++ )
+  {
+    QListWidgetItem *item = mFieldsList->item( i );
+
+    if ( item->isSelected() )
+    {
+      if ( item->checkState() == Qt::Unchecked )
```

Comment:
```suggestion
      item->setCheckState( item->checkState() == Qt::Checked ? Qt::Unchecked :  Qt::Checked  );
```
Not mandatory but it looks more concise IMHO